indico/modules/designer/models/templates.py

```diff
@@ -89,11 +89,11 @@ class DesignerTemplate(db.Model):
             lazy=True
         )
     )
-    background_image = db.relationship(
-        'DesignerImageFile',
+    backside_template = db.relationship(
+        'DesignerTemplate',
         lazy=True,
-        foreign_keys=background_image_id,
-        post_update=True
+        foreign_keys=backside_template_id,
+        remote_side=id
     )
```

Comment:
Why do we need to remove `background_image` here? Didn't we agree things would stay the same, we'd only add a reference to a backside?